log-collector-script/linux/eks-ssm-content.json

```diff
@@ -21,17 +21,17 @@
             "runCommand": [
                "curl -O https://raw.githubusercontent.com/awslabs/amazon-eks-ami/master/log-collector-script/linux/eks-log-collector.sh",
                "echo \"Cleaning old eks-log-collector files\"",
-               "rm /var/log/eks_i*",
+               "rm -f /var/log/eks_i*",
                "bash ./eks-log-collector.sh  >/dev/null 2>&1",
                "echo \"EKS logs collected\"",
                "if [ -f /usr/local/bin/aws ]; then",
                "echo \"AWS_already_installed\"",
                "else",
-               "echo \"Installing AWSCLI\"",
-               "curl \"https://s3.amazonaws.com/aws-cli/awscli-bundle.zip\" -o \"awscli-bundle.zip\"  >/dev/null 2>&1",
+               "echo \"Installing AWSCLI v2\"",
+               "curl \"https://awscli.amazonaws.com/awscli-exe-linux-x86_64.zip\" -o \"awscliv2.zip\"  >/dev/null 2>&1",
```

Comment:
The `ARCH` may not always be `x86_64`, can you add handling for that?

Reply:
Modified

Reply:
@cartermckinnon 
Could you please re-check it?